c10/core/DispatchKey.h

```diff
@@ -222,15 +222,16 @@ enum class DispatchKey : uint8_t {
   // you are responsible for handling autograd yourself, or deferring to other
   // operators which support autograd.
 
-  // Currently we only have backend-specific autograd keys for CPU/CUDA/XLA and
-  // reserved user-defined backends. All other in-tree backends share the
+  // Currently we only have backend-specific autograd keys for CPU/CUDA/XLA/ORT
+  // and reserved user-defined backends. All other in-tree backends share the
   // AutogradOther key. We can add specific autograd key for those backends
   // upon request.
   AutogradOther,
   AutogradCPU,
   AutogradCUDA,
   AutogradXLA,
   AutogradLazy,
+  AutogradORT,
```

Comment:
Do you have an example of how you're going to use that key?

Reply:
Not at the moment. I've tabled the work on the ORT side that was investigating this path for now, so I can drop it from this PR. We can revisit in another iteration if we need it or something similar.